akka-actor/src/main/scala/akka/dispatch/AbstractDispatcher.scala

```diff
@@ -23,7 +23,7 @@ import akka.event.EventStream
 import akka.event.Logging.{ Debug, Error, LogEventException }
 import akka.util.{ unused, Index, Unsafe }
 
-final case class Envelope private (val message: Any, val sender: ActorRef)
+final case class Envelope private (message: Any, sender: ActorRef)
```

Comment:
I have compared the change with `:javap -pc akka.dispatch.Envelope` and the byte code and signatures are identical.